src/main/java/com/olehedza/reviews/util/CsvParser.java

```diff
@@ -0,0 +1,63 @@
+package com.olehedza.reviews.util;
+
+import com.olehedza.reviews.dto.parser.CsvDto;
+import java.io.StringReader;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVRecord;
+import org.springframework.stereotype.Component;
+
+@Component
+@Slf4j
+public class CsvParser implements Parser<CsvDto> {
+    private final List<CsvDto> dtos = new ArrayList<>();
+    private final FileReader fileReader;
+
+    public CsvParser(FileReader fileReader) {
+        this.fileReader = fileReader;
+    }
+
+    @SneakyThrows
+    @Override
+    public List<CsvDto> parse(String path) {
+        StringBuilder stringBuilder = new StringBuilder();
+        for (String string : fileReader.readFile(path)) {
+            stringBuilder
+                    .append(string)
+                    .append("\n");
```

Comment:
```suggestion
            stringBuilder.append(string).append("\n");
```
можливо є інший спосіб за допомогою якого можна парсити відразу List<String>?

Reply:
Можливо він і є, але я поки не знаю. Я витратив і так багато часу на пошук оптимальних варіантів парсингу із заголовками. Цей спосіб далеко не самий продуктивний, але робочий.